backend/frontend/templates/frontend/user/board/partial_card.html

```diff
@@ -207,12 +207,9 @@ <h5 class="w-[auto] h-[auto] text-black text-base font-bold font-'Poppins' leadi
                   hx-swap="outerHTML">Cancel review request</button>
         {% endif %}
       </div>
-
-      <div class="grid grid-cols-2 grid-auto-rows-1fr gap-x-10 gap-y-10 w-full h-[auto]">
-        <div class="col-span-1 row-span-1 w-full">
+      {%if card.content_item.content_type_nice == "project" and card.content_item.project_submission_type_nice == "link" %}
```

Comment:
This will need to be fixed again as soon as the topic details page is in place.

Why not rather fix it in a perminant way?

You can make placeholder views for the different types of content and then the link can go there. And then the actual pages can get implemented later.

If you fix something in a way that means you need to fix it again later, is it really fixed?  Let's steer clear of duct-tape